.github/scripts/run-bisection.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -xeo pipefail
+
+SCRIPTPATH=$(realpath $0)
+BASEDIR=$(dirname $SCRIPTPATH)
+BISECT_BASE=${HOME}/bisection/gh${GITHUB_RUN_ID}
+
+BISECT_CONFIG=${BISECT_BASE}/config.env
+set -a
+source $BISECT_CONFIG
+set +a
+
+# get torch_nightly.html
+curl -O https://download.pytorch.org/whl/nightly/cu102/torch_nightly.html 
+
+. activate ${BISECT_CONDA_ENV} &> /dev/null
```

Comment:
can you better-document the intended API of this script at the top? e.g. which env vars are supposed to be set for it to work, and what args to provide on CLI?

Reply:
Now it requires only one external file to work: the "config.yaml". I have added documentations to the file header.